### PR TITLE
Upgrade support for Distant Horizons 2.2

### PR DESCRIPTION
**Pull request by IMS212**

### CODE REVIEW

File: src/main/java/net/irisshaders/iris/mixin/MixinLevelRenderer.java [hunk 1]
```diff
@@ -32,6 +32,8 @@
 import org.spongepowered.asm.mixin.injection.*;
```
Comment (dima-dencep) [inline src/main/java/net/irisshaders/iris/mixin/MixinLevelRenderer.java]: Revert

File: src/main/java/net/irisshaders/iris/compat/dh/DHMixinConfigPlugin.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	@Override
+	public void preApply(String targetClassName, ClassNode targetClass, String mixinClassName, IMixinInfo mixinInfo) {
+
+	}
+
+	@Override
+	public void postApply(String targetClassName, ClassNode targetClass, String mixinClassName, IMixinInfo mixinInfo) {
+
+	}
+}
```
Comment (dima-dencep) [inline src/main/java/net/irisshaders/iris/compat/dh/DHMixinConfigPlugin.java]: Such a config plugin already exists

File: build.gradle [hunk 3]
```diff
@@ -110,7 +110,7 @@ dependencies {
 	forge "net.neoforged:forge:${minecraft_version}-${forge_version}"
 
 	modImplementation "org.embeddedt:embeddium-${minecraft_version}:${embeddium_version}"
-	modCompileOnly(files("DistantHorizons-forge-2.0.4-a-dev-1.20.1.jar"))
+	modCompileOnly("maven.modrinth:distanthorizonsapi:3.0.0")
```
Comment (dima-dencep) [inline build.gradle:113]: Maybe the file itself should be deleted too?

**Review by dima-dencep — CHANGES_REQUESTED** [review]

**Comment by IMS212** [comment]
done

**Review by dima-dencep — APPROVED** [review]

**Comment by mpustovoi** [comment]
Any further progress here?

**Pull request merged**